lacework/resource_lacework_managed_policies.go

```diff
@@ -0,0 +1,153 @@
+
+		Schema: map[string]*schema.Schema{
+			"policy": {
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "The id of the policy",
+							Required:    true,
+						},
+						"enabled": {
+							Type:        schema.TypeBool,
+							Description: "The state of the policy",
+							Required:    true,
+						},
+						"severity": {
+							Type:     schema.TypeString,
+							Required: true,
+							Description: "The severity for the policy. Valid severities are: " +
+								"Critical, High, Medium, Low, Info",
+							StateFunc: func(val interface{}) string {
+								return strings.TrimSpace(strings.ToLower(val.(string)))
+							},
+							ValidateDiagFunc: ValidSeverity(),
+						},
+					},
+				},
+				Required:    true,
+				Description: "A list of Lacework managed policies",
+			},
+		},
+	}
+}
+
+func resourceLaceworkManagedPoliciesCreate(d *schema.ResourceData, meta interface{}) error {
+	d.SetId(time.Now().UTC().String())
+	return resourceLaceworkManagedPoliciesUpdate(d, meta)
+}
+
+func resourceLaceworkManagedPoliciesUpdate(d *schema.ResourceData, meta interface{}) error {
+	lacework := meta.(*api.Client)
+	policies, err := getBulkUpdatePolicies(d)
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[INFO] Updating Policies with data:\n%+v\n", policies)
+	_, updateErr := lacework.V2.Policy.UpdateMany(policies)
+	if updateErr != nil {
+		return updateErr
+	}
+	log.Printf("[INFO] Updated Policies with data:\n%+v\n", policies)
+	return nil
+}
+
+func resourceLaceworkManagedPoliciesRead(d *schema.ResourceData, meta interface{}) error {
+	lacework := meta.(*api.Client)
+
+	policiesListResponse, err := lacework.V2.Policy.List()
+
+	if err != nil {
+		return err
+	}
+
+	bulkUpdatePolicies, err := getBulkUpdatePolicies(d)
+
+	if err != nil {
+		// Return nil so that `destroy` can succeed
+		return nil
+	}
+
+	policyMap := make(map[string]api.Policy, len(policiesListResponse.Data))
+
+	for _, policy := range policiesListResponse.Data {
+		policyMap[policy.PolicyID] = policy
+	}
+
+	policySet := make([]map[string]any, len(bulkUpdatePolicies))
+
+	for i, bulkUpdatePolicy := range bulkUpdatePolicies {
+		policySet[i] = map[string]any{}
+		policySet[i]["id"] = bulkUpdatePolicy.PolicyID
+		policySet[i]["enabled"] = policyMap[bulkUpdatePolicy.PolicyID].Enabled
+		policySet[i]["severity"] = policyMap[bulkUpdatePolicy.PolicyID].Severity
+	}
+
+	d.Set("policy", policySet)
+
+	return nil
+}
+
+func resourceLaceworkManagedPoliciesDelete(d *schema.ResourceData, meta interface{}) error {
+	d.SetId("")
+	return nil
+}
+
+func getBulkUpdatePolicies(d *schema.ResourceData) (api.BulkUpdatePolicies, error) {
+	var policies api.BulkUpdatePolicies
+	list := d.Get("policy").(*schema.Set).List()
+	seen := make(map[string]bool, 0)
+
+	for _, v := range list {
+		val := v.(map[string]interface{})
+
+		if val["id"] == nil || val["id"] == "" {
```

Comment:
can `id`s be nil or empty? 🤔 

I feel we need to unit test this code

Reply:
When running the tests, entries with empty `id` are being added for some reason.

For example:
<img width="1027" alt="Screenshot 2023-07-24 at 3 12 25 PM" src="https://github.com/lacework/terraform-provider-lacework/assets/7945666/602cce6d-970b-448a-8415-8d68269a5bf1">

Could not figure out why this is happening. Any ideas?